dask/base.py

```diff
@@ -707,7 +716,7 @@ def visualize(
         try:
             cmap = kwargs.pop("cmap")
         except KeyError:
-            cmap = plt.cm.RdBu
+            cmap = plt.cm.plasma
```

Comment:
Happy to roll this one back, but I thought it was more appropriate to use a sequential colormap rather than a diverging one for all the currently implemented coloring schemes

Reply:
+1 for a sequential colormap